compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/node/NodeChain.kt

```diff
@@ -828,4 +839,21 @@ private fun Modifier.fillVector(
         }
     }
     return result
-}
\ No newline at end of file
+}
+
+@Suppress("ModifierNodeInspectableProperties")
+private object ForceUpdateElement : ModifierNodeElement<Modifier.Node>() {
+    override fun create(): Modifier.Node {
+        throw IllegalStateException("Shouldn't be called")
+    }
+
+    override fun update(node: Modifier.Node) {
+        throw IllegalStateException("Shouldn't be called")
+    }
+
+    override fun hashCode(): Int = 100
```

Comment:
Is there a reason why it equals 100? 
I know that may be a constant if class is singleton, but I don't quite understand why 100.

Reply:
Not sure why :), it was done in the Jetpack Compose PR.

This code works, but usually we don't need to override hashCode for `object`.